fastlane/Fastfile

```
@@ -0,0 +1,65 @@
+      groups: "#{ENV['TESTER_GROUPS']}",
+      release_notes: "#{options[:variant]}-Debug version out!",
+      debug: true
+    )
+  end
+
+  desc "Build, Sign, and Distribute Release APK to Play Store"
+  lane :build_and_distribute_release do |options|
+    # Build the release variant
+     puts "variant: #{options[:variant]} ^& bundle#{options[:variant]}Release"
+    gradle(
+          task: "bundleNiramayRelease",
```

rubocop (1.69.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)